<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nameBox_2</class>
 <widget class="QMainWindow" name="nameBox_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(38, 38, 38);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ff8177;</string>
    </property>
    <property name="text">
     <string>Manage URLs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QComboBox" name="namesBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
border: 2px solid rgb(66, 64, 64);
font-size:15pt;
color: 	rgb(136,136,136);
text-align: center;
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Source name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="urlBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
border: 2px solid rgb(66, 64, 64);
font-size:15pt;
color: 	rgb(136,136,136);
text-align: center;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addUrlButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #f77062;
color: #fff;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Add URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #f77062;
color: #fff;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Remove URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #f77062;
color: #fff;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Go back</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:14pt;
color: #f77062;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="saveChangesButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #f77062;
color: #fff;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Save changes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
